app/uk/gov/hmrc/apinotificationqueue/controller/QueueController.scala

```diff
@@ -17,20 +17,38 @@
 package uk.gov.hmrc.apinotificationqueue.controller
 
 import java.util.UUID
-import javax.inject.Singleton
+import javax.inject.{Inject, Singleton}
 
+import org.joda.time.DateTime
 import play.api.http.HttpEntity
 import play.api.mvc._
+import uk.gov.hmrc.apinotificationqueue.repository.Message
+import uk.gov.hmrc.apinotificationqueue.service.QueueService
+import uk.gov.hmrc.http.BadRequestException
 import uk.gov.hmrc.play.bootstrap.controller.BaseController
 
 import scala.concurrent.Future
 
 @Singleton()
-class QueueController extends BaseController {
+class QueueController @Inject()(queueService: QueueService) extends BaseController {
+  val CLIENT_ID = "x-client-id"
 
```

Comment:
note only: maybe worth extracting this to an object containing header names soon - eg `CLIENT_ID_HEADER_NAME`